platform/accessibility/common-issues/focus-order-guidelines.md

```diff
@@ -0,0 +1,70 @@
+# [KEYBOARD]: Focus order MUST be visually and programmatically usable
+We build a lot of rich interet applications (RIA) using React, for VA.gov. We often update routes using client-side technology, add and remove HTML fragments, and update interfaces without making page requests to a server.
+
+These type of client-side changes require us to think about focus order. 
+
+- What elements can receive keyboard focus when users press the `TAB` key or the `SHIFT + TAB` chord?
+- Are there non-focusable elements that we want to receive keyboard focus?
+- Are there elements that need to receive focus when a dynamic event like a data fetch is completed?
+
+This is a common issue that can be solved with human-centered design and a collaborative effort among design, front-end engineering, and accessibility.
```

Comment:
recommendation: invert this section to state the issue first and then the details about this issue